### PR TITLE
apparmor: update to 3.0.1

### PR DESCRIPTION
**Pull request by paper42**

### CODE REVIEW

File: srcpkgs/apparmor/template [hunk 1]
```diff
@@ -15,8 +15,9 @@ short_desc="Mandatory access control to restrict programs"
 maintainer="Olivier Mauras <olivier@mauras.ch>"
 license="GPL-2.0-only, LGPL-2.1-only"
 homepage="https://gitlab.com/apparmor/apparmor"
+changelog="https://gitlab.com/apparmor/apparmor/-/wikis/Release_Notes_3.0.1"
```
Comment (ericonr) [inline srcpkgs/apparmor/template:18]: `${version}`

Also wow having versioned release notes sucks :/

Comment (ericonr) [reply srcpkgs/apparmor/template:18]: Hmm, in this case it's having to update this manually vs having the URL not have variables.....

Comment (paper42) [reply srcpkgs/apparmor/template:18]: I missed that, thanks. There is a changelog file in libraries/libapparmor/ChangeLog, but... it hasn't been updated in 12 years and it's an empty file. :)

**Comment by ericonr** [comment]
I'm going to reboot into it and merge then.

**Pull request merged**